pkgs/development/libraries/opencascade/default.nix

```diff
@@ -1,6 +1,13 @@
 { stdenv, fetchFromGitHub, fetchpatch, libGL, libGLU, libXmu, cmake, ninja,
-  pkgconfig, fontconfig, freetype, expat, freeimage, vtk }:
+  pkgconfig, fontconfig, freetype, expat, freeimage, vtk
 
+, withGl2ps ? false, gl2ps
```

Comment:
My personal opinion is that we should avoid adding too much configuration, unless it is: a) used by other packages in nixpkgs b) built (means tested) on Hydra. In this case, it is seems to be perfectly reasonable to add the small `gl2ps` dependency unconditionally. The `tbb` is a bigger dependency, and without knowing much about opencascade I can't tell if some people would ever want a build without `tbb`. I believe, in nixpkgs we generally prefer to build against most comprehensive set of dependencies, so `tbb` should be included by default too. That way, the people who do want it, they would be able to get a build substituted from the binary cache.

Reply:
I didn't know opencascade existed before this ;)
Wikipedia says it's an SDK for CAD/CAM/EDA/… afaikt Opencascade offers two multithreading options: TBB and OpenMP. The build script for KiCad, which I looked at enabled TBB, so I chose that one too. No deeper reason on my part.
But I will enable both per default.

Reply:
I think it makes sense to drop the options altogether.

Reply:
Oh, ok. I misunderstood that.